/**:
  ros__parameters:
    min_points_num:
    # CAR     TRUCK BUS  BICYCLE   PEDESTRIAN
      [10,     10,  10,    10,         10]

    max_points_num:
    # CAR     TRUCK BUS  BICYCLE   PEDESTRIAN
      [100,     100,  100,    10,         10]

    min_points_and_distance_ratio:
    # CAR     TRUCK     BUS     BICYCLE   PEDESTRIAN
      [800.0,     880.0,     800.0,    800.0,         800.0]
